impl/cluster/src/main/java/org/pentaho/big/data/impl/cluster/NamedClusterManager.java

```diff
@@ -348,10 +348,17 @@ private String getSlaveServerMetastoreDir() throws IOException {
         // at once, and one of them is packaging up the install for a yarn carte job
         && KettleClientEnvironment.getInstance().getClient().equals( KettleClientEnvironment.ClientType.CARTE ) ) {
         return slaveMetaStorePath;
+      }
+
+      slaveMetaStorePath = System.getProperty( "user.home" ) + File.separator + ".pentaho";
```

Comment:
Do we want to look for the metastore folder specifically in here?  Or is .pentaho sufficient?

Reply:
It’s sufficient to find just .pentaho. The code automatically looks for a metastore folder in that location.